scripts/write-emails-to-disk.js

```diff
@@ -124,7 +124,10 @@ function getMailerMessageTypes(mailer) {
   var messageTypes = []
 
   for (var key in mailer) {
-    if (typeof mailer[key] === 'function' && ! /^_/.test(key) && /Email$/.test(key)) {
+    if (
+      typeof mailer[key] === 'function' &&
+      ! /^_/.test(key) && ! /^send/.test(key) && /Email$/.test(key)
```

Comment:
I made this fix in a bit of a rush, but perhaps a more robust approach would be better overall. Instead of trying to magically guess the correct methods to invoke, should this script just contain a hard-coded list of method names instead?